ext/standard/math.c

```diff
@@ -170,56 +122,30 @@ static inline double php_round_helper(double value, int mode) {
  * mode. For the specifics of the algorithm, see http://wiki.php.net/rfc/rounding
  */
 PHPAPI double _php_math_round(double value, int places, int mode) {
-	double f1, f2;
+	double f1;
 	double tmp_value;
-	int precision_places;
 
 	if (!zend_finite(value) || value == 0.0) {
 		return value;
 	}
 
 	places = places < INT_MIN+1 ? INT_MIN+1 : places;
```

Comment:
This is unrelated to this, but possibly something we should adress in a follow-up PR, to emit a warning or throw a ValueError if the provided places argument doesn't make sense